build.gradle

```diff
@@ -72,4 +75,4 @@ test {
 
```

Comment:
do you need a classpath/runtime dependency on error-prone as well?

https://github.com/google/error-prone/blob/master/examples/gradle/build.gradle#L9